snapshots/snapshots/src/main/kotlin/com/emergetools/snapshots/SnapshotImageMetadata.kt

```diff
@@ -5,10 +5,12 @@ import kotlinx.serialization.ExperimentalSerializationApi
 import kotlinx.serialization.Serializable
 import kotlinx.serialization.json.JsonClassDiscriminator
 
+@Deprecated(
+  "Composable is the only supported view type. " +
+    "For legacy views, wrap with AndroidView"
+)
 enum class SnapshotType {
```

Comment:
Should we kill this enum too, or worth leaving around? Removing cases is a backwards-incompatible change so might as well kill the enum while we're at it IMO.

Reply:
This will require backend changes to accomodate, but I'll look into removing before merging this PR.

Reply:
Backend changes done and merged, will be landing this soon!